port : $(arg driver_port)
baud : 115200

frequency_list: [10] #higher frequency (value is in ms)

channel_mode: dual
# motor_type:  set_speed
motor_1_type: set_speed
motor_2_type: set_speed


query0:
 motor_amps : ?A 
 motor_command : ?M
 hall_count : ?BCR
 abs_hall_count : ?CB
 hall_speed : ?BS
 volts : ?V
 feedback : ?F 
 battery_amps : ?BA
 power : ?P          
 fault_flag : ?FF  
 sto_status : ?STT 
#  motor_command : ?M
#  volts : ?V
#  feedback : ?F 
#  battery_amps : ?BA
#  power : ?P          
#  fault_flag : ?FF  
# motor_amps2 : ?MA
# motor_command2 : ?FM
# hall_count2 : ?FM
# hall_speed2 : ?FS
# volts2 : ?V
# feedback2 : ?rmp 
# battery_amps2 : ?PHA
# power2 : ?sns          
# fault_flag2 : ?S  
# status_flag : ?FS
# firmware_id : ?FID  
# encoder_count : ?C
# encoder_speed : ?S  